packages/cashier/src/components/alert-banner/alert-banner.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import './alert-banner.scss';
+
+type TAlertBanner = {
+    className?: string;
+    icon: string;
+    message: JSX.Element | string;
```

Comment:
@heorhi-deriv suggested me to use `<Localize />` component instead of the function as he pointed out that the function variant is used only for simple strings and not for dynamic data throughout the project.